### ISSUE TITLE
Remove cocotb dependency from Workshop

### ISSUE THREAD

**Issue by pcesar22** [SpinalHDL/SpinalWorkshop#7]
Cocotb is very annoying to set up and use, and SpinalSim should be the default way to set up unit tests.

**Comment by typingArtist** [comment]
Good idea, this would solve a plethora of issues. We just need someone to rewrite the respective tests in SpinalSim.
